feature_engine/dataframe_checks.py

```diff
@@ -29,8 +29,9 @@ def _is_dataframe(X: pd.DataFrame) -> pd.DataFrame:
     """
```

Comment:
could you please remove this file from the PR.

This change is causing all tests to fail. There is no reason why this file should be changed.